jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -997,6 +997,17 @@ public static void load(String pathName) {
  */
 @CallerSensitive
 public static void loadLibrary(String libName) {
+	if (libName.indexOf(File.pathSeparator) >= 0) {
+		/*[MSG "K0B01", "Library name must not contain a file path: {0}]*/
+		throw new UnsatisfiedLinkError(com.ibm.oti.util.Msg.getString("K0B01", libName)); //$NON-NLS-1$
```

Comment:
```suggestion
		throw new UnsatisfiedLinkError(Msg.getString("K0B01", libName)); //$NON-NLS-1$
```
There are quite a few `com.ibm.oti.util.Msg` references which can be eliminated via an `import`.

Reply:
I'll do this separately after everything else is resolved.

Reply:
> I'll do the import separately after everything else is resolved.

Were you planning to do that in this pull request or a separate one?

Reply:
I've added the commit.